client/document/index.jsx

```diff
@@ -79,6 +80,9 @@ class Document extends React.Component {
 			'jetpack-connect' === sectionName &&
 			'woocommerce-onboarding' === requestFrom;
 
+		const isJetpackCloudFlow =
```

Comment:
Nitpicking: I'd suggest `section` vs `flow`, as Jetpack Cloud isn't really "flow".

```suggestion
		const isJetpackCloudSection
```